tests/trestlebot/cli/test_sync_cac_content_cmd.py

```diff
@@ -118,13 +118,19 @@ def test_sync_product(tmp_repo: Tuple[str, Repo]) -> None:
     component = compdef.components[0]
     assert component.title == "rhel8"
     # Check rules component props
-    assert len(component.props) == 6
+    assert len(component.props) == 24
```

Comment:
LGTM.  I tested it with test data in tests/data/content_dir, and it works.  After resolving the conflicts, the line175 needs to be updated to `assert len(component.props) == 30`.

Reply:
Um, I checked the parameters properties, two Paramter_Id could have the same remarks, I am not sure if this is the expected result. @marcusburghardt do you have any comments?

Reply:
One rule can use multiple variables and one variable can be used by multiple rules. But in the context of a profile, a variable has the same option/value. I am checking the PR now.